blockmanager.go

```diff
@@ -1561,10 +1591,10 @@ func (b *blockManager) handleInvMsg(imsg *invMsg) {
 	}
 }
 
-// limitMap is a helper function for maps that require a maximum limit by
-// evicting a random transaction if adding a new value would cause it to
+// addAndLimitMap is a helper function for maps that require a maximum limit by
+// evicting a random transaction if adding the new value would cause it to
```

Comment:
Should probably use the generic term `value` here  instead of `transaction` since `addAndLimitMap` is used both tx and block hash maps.

Reply:
Would `limitAdd` make a better choice for this func name?